### PR TITLE
Fix/switch labeled test 133

### PR DESCRIPTION
**Pull request by albertoestarrona**
> Please check you are not duplicating an existing pull request...

#### Changes
Test for #133

### CODE REVIEW

File: __tests__/src/components/common/SwitchLabeled.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import SwitchLabeled from '../../../../src/components/common/SwitchLabeled';
+import React from 'react';
+import renderer from 'react-test-renderer'
+import Strings from "../../../../src/global/Strings";
+
+test('SwitchLabeled renders correctly', () => {
```
Comment (seland) [inline __tests__/src/components/common/SwitchLabeled.js]: You can use [describe](https://facebook.github.io/jest/docs/en/api.html#describename-fn) method to compose tests for similar purpose (optional).

File: __tests__/src/components/common/SwitchLabeled.js [hunk 2]
```diff
@@ -1,23 +1,54 @@
...
+  describe('SwitchLabeled behaviour', () => {
+
+    test('SwitchLabeled setup to be true', () => {
+      const tree = renderer.create(
+        <SwitchLabeled
+          label={'Testing SwitchLabeled'}
+          value={true}
+        />
+      ).toJSON();
+      expect(tree).toBeTruthy();
```
Comment (seland) [inline __tests__/src/components/common/SwitchLabeled.js]: What is that supposed to do? Should it be that tree equals true?

**Comment by albertoestarrona** [comment]
@seland Added the test for value change, but have problem to check the new value test

**Comment by albertoestarrona** [comment]
@seland The doc for the props was already there. Tell me if need to be more explicit

**Pull request merged**